dask_expr/_collection.py

```diff
@@ -2649,3 +2731,41 @@ def isna(arg):
         return arg.isna()
     else:
         return from_pandas(arg).isna()
+
+
+def elemwise(op, *args, meta=no_default, out=None, transform_divisions=True, **kwargs):
+    """Elementwise operation for Dask dataframes
+
+    Parameters
+    ----------
+    op: callable
+        Function to apply across input dataframes
+    *args: DataFrames, Series, Scalars, Arrays,
+        The arguments of the operation
+    meta: pd.DataFrame, pd.Series (optional)
+        Valid metadata for the operation.  Will evaluate on a small piece of
+        data if not provided.
+    transform_divisions: boolean
+        If the input is a ``dask.dataframe.Index`` we normally will also apply
+        the function onto the divisions and apply those transformed divisions
+        to the output.  You can pass ``transform_divisions=False`` to override
+        this behavior
+    out : ``dask.array`` or ``None``
+        If out is a dask.DataFrame, dask.Series or dask.Scalar then
+        this overwrites the contents of it with the result
+    **kwargs: scalars
+
+    Examples
+    --------
+    >>> elemwise(operator.add, df.x, df.y)  # doctest: +SKIP
+    """
+
+    args = _maybe_from_pandas(args)
+
+    # TODO: Add align partitions
+
+    dfs = [df for df in args if isinstance(df, FrameBase)]
+
+    return new_collection(
+        expr.UFuncElemwise(dfs[0], op, meta, transform_divisions, kwargs, *args)
+    )
```

Comment:
It doesn't need `*args[:-1]`? 
```suggestion
    df = next(df for df in args if isinstance(df, FrameBase))

    return new_collection(
        expr.UFuncElemwise(df, op, meta, transform_divisions, kwargs, *args[:-1])
    )
```

Reply:
This would change argument order, which is not good when we hand them over to the actual method, the current pattern isn't great but it's probably the best we can do